exercises/exercise3.js

```diff
@@ -1,12 +1,419 @@
+    return 'I can help you identify Native Australian plants... '
+    + '\nLike the Native Hydrangea, aka ' + name 
+    + ', which is native to New South Wales and Queensland...'
+    + '\n' + link;
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'What are you designed to do',
+    prompt: 'What do you do then?',
+    handler: function() {
+      var name = 'Acacia filicifolia';
+      var link = 'https://en.wikipedia.org/wiki/Acacia_filicifolia';
+    return 'I help identify Native Australian plants... '
+    + '\nLike the common Native Fern-Leaved Wattle, aka ' + name 
+    + ' whihc is widespread across the New South Wales Tablelands...'
+    + '\n' + link;
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Bot explained script 0',
+    prompt: 'Okay, how do you do that?',
+    handler: function() {
+      return 'Tell me where you are by typing \'My location is...\' and I\'ll tell you about the Native plants in the area...'
+      + '\n(right now I\'m still bound to location only... but in the future I\'ll be able to identify plants based on things like petal shape, location, colour and seeds)';
+      }, 
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Bot explained script 1',
+    prompt: 'Okay, how do you do that then?',
+    handler: function() {
+      return 'ASL....'
+      + '\nJust kidding! That\'d be gross'
+      + '\nTell me where you are by typing \'Okay, my location is...\'and I\'ll tell you about the Native plants in the area...'
+      + '\n(right now I\'m still bound to location only... but in the future I\'ll be able to identify plants based on things like petal shape, location, colour and seeds)';
+      },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Bot explained script 2',
+    prompt: 'How?',
+    handler: function() {
+      return 'Tell me where you are by typing \'My location is...\' and I\'ll tell you about the Native plants in the area...'
+      + '\n(right now I\'m still bound to location only... but in the future I\'ll be able to identify plants based on things like petal shape, location, colour and seeds)';
+    },
+    // isReply: true,
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Location checking script',
+    prompt: 'Okay, my location is #{location}',
+    handler: function(params) {
+      if (params.location.toLowerCase() === 'nsw' || 
+          params.location.toLowerCase() === 'new south wales') {
+        return 'The most common plants in ' + params.location + 'are ...';
+      } else {
+        return ' I\'m sorry I couldnt find any plants in that area, maybe try the region?';
+        }
+    },
+  },
+  {
+    label: 'Location checking script',
+    prompt: 'My location is #{location}',
+    handler: function(params) {
+      if (params.location.includes(plants.locations)) {
```

Comment:
So close on this one! The rest of your code is in the right place, but first instead of 'plants.locations' you need to iterate through the objects you created for the variable 'plants'. I'd suggest looking into Objects.keys and Objects.values or the use of a for/in loop too.
Because these are all strings, you can utilize the '.includes()', but right now plants.locations isn't going to return anything as you haven't traversed/iterated through your object yet.